### ISSUE TITLE
A11y_Visual Studio Code_Docs Setup_Bypass Blocks: 'Skip to content' link is not visible on the screen

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [microsoft/vscode-docs#4524]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags:#A11y_Visual Studio Code#A11ySev2#A11yTCS#A11yMAS#Chrome#DesktopWeb#Win10#Visual Studio Code Services#WCAG2.4.1#Bypass Blocks#VisualStudioCode-Web-Apr2021

### Environment details:
Application Name: Visual Studio Code
URL:[Visual Studio Code](https://code.visualstudio.com/docs/setup/linux)
Windows Version: Version 20H2 (OS Build 19042.928)
Browser: Chrome Version 90.0.4430.85 (Official Build) (64-bit)

### Note:
This issue is observed through out the application

### Repro Steps:

1. Hit [Visual Studio Code](https://code.visualstudio.com/docs/setup/linux) link
2. Tab till skip link and verify whether skip link is visible when focus lands on it or not

### Actual Result:
'Skip to content' link is not visible on the screen

### Expected Result:
'Skip to content' link should be clearly visible on the screen when focus lands on it.

### User Impact:
Keyboard users may get confuse and may not understand that skip link is available on screen if it is not visible

### Attachment:

https://user-images.githubusercontent.com/83570638/117128074-91688300-adba-11eb-89ea-7252aece9c08.mp4

**Comment by gregvanl** [comment]
Fixed

**Comment by kalpanah1** [comment]
#Closed;
Verified the issue in the below environment. The issue is fixed. Please find the attachment for reference.
URL: https://code.visualstudio.com.
![Bug_4524_Fixed](https://user-images.githubusercontent.com/94966876/217816753-f6f7409a-237f-4be9-9c6d-8a689d1e72f5.gif)

**Comment by kalpanah1** [comment]
Adding the attachment.
[Uploading Bug_4524_Fixed.zip…]()
